src/haz3lweb/Keyboard.re

```diff
@@ -97,10 +97,25 @@ let handle_key_event = (k: Key.t): option(Action.t) => {
     | _ => None
     }
   | {key: D("f"), sys: PC, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
-    Some(Project(ToggleIndicated(Fold)))
+    Some(Project(SetIndicated(Specific(Fold))))
   | {key: D("ƒ"), sys: Mac, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
     /* Curly ƒ is what holding option turns f into on Mac */
-    Some(Project(ToggleIndicated(Fold)))
+    Some(Project(SetIndicated(Specific(Fold))))
+  | {key: D("v"), sys: PC, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    Some(Project(SetIndicated(Specific(Probe))))
+  | {key: D("√"), sys: Mac, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    /* √ is what holding option turns f into on Mac */
+    Some(Project(SetIndicated(Specific(Probe))))
+  | {key: D("t"), sys: PC, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    Some(Project(SetIndicated(Specific(Info))))
+  | {key: D("†"), sys: Mac, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    /* † is what holding option turns t into on Mac */
+    Some(Project(SetIndicated(Specific(Info))))
+  | {key: D("l"), sys: PC, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    Some(Project(SetIndicated(ChooseLivelit)))
+  | {key: D("¬"), sys: Mac, shift: Up, meta: Up, ctrl: Up, alt: Down} =>
+    /* † is what holding option turns t into on Mac */
+    Some(Project(SetIndicated(ChooseLivelit)))
```

Comment:
Would you be open to putting some of these in https://github.com/hazelgrove/hazel/blob/dev/src/haz3lweb/app/input/Shortcut.re so that way we can trigger them from the command palette as well?

Reply:
i'll add these as a separate PR later as ninjakeys is down